src/features/feed/components/FreshSection.jsx

```diff
@@ -66,11 +66,12 @@ const FreshSection = ({
   //   setCategories([...searchResult]);
   // }, [searches]);
 
-  if (bukkaMenu.length === 1 && error) {
+  // if (bukkaMenu.length === 1 && error) {
```

Comment:
Why do you comment out this line? Please add it back. It should only show when there is an error.

Reply:
resolved